subprojects/solver/solver-smtlib/src/main/java/hu/bme/mit/theta/solver/smtlib/impl/smtinterpol/SMTInterpolSmtLibSolverInstaller.java

```diff
@@ -81,34 +81,23 @@ public SolverFactory getSolverFactory(final Path installDir, final String versio
 
     @Override
     public List<String> getSupportedVersions() {
-        return Arrays.asList("2.5-1256", "2.5-1230", "2.5-916", "2.5-663", "2.5-479", "2.5-7");
+        return Arrays.asList("2.5-1301", "2.5-1256", "2.5-1230", "2.5-916", "2.5-663", "2.5-479", "2.5-7");
     }
 
     private URL getDownloadUrl(final String version)
             throws SmtLibSolverInstallerException, MalformedURLException {
-        final String fileName;
-        switch (version) {
-            case "2.5-1256":
-                fileName = "2.5-1256-g55d6ba76";
-                break;
-            case "2.5-1230":
-                fileName = "2.5-1230-g3eafb46a";
-                break;
-            case "2.5-916":
-                fileName = "2.5-916-ga5843d8b";
-                break;
-            case "2.5-663":
-                fileName = "2.5-663-gf15aa217";
-                break;
-            case "2.5-479":
-                fileName = "2.5-479-ga49e50b1";
-                break;
-            case "2.5-7":
-                fileName = "2.5-7-g64ec65d";
-                break;
-            default:
-                throw new SmtLibSolverInstallerException("Unsupported solver version.");
-        }
+        final String fileName = switch (version) {
+
+            case "2.5-1256" -> "2.5-1256-g55d6ba76";
+            //TODO download won't work on this, switch to the latest on next release
+            case "2.5-1301" -> "2.5-1301-g2c871e40";
```

Comment:
Please move 1301 higher so they are in order

Reply:
No, since officially that release does not exist yet, so to have smtinterpol:latest solver string work, the order has to be this. The comment refers to that issue